src/serverHome/home.html

```diff
@@ -0,0 +1,1223 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1>custom options page</h1>
```

Comment:
i didn't get the utility of this folder cause your server.js file is at the root level

Reply:
it was only for experimenting with the DOM selectors and just to see if everything works fine while injecting the script on a similarly structured HTML page